en/activity-standard/iati-activities/iati-activity/transaction/recipient-region.rst

```diff
@@ -25,6 +25,10 @@ Full example:
 Changelog
 ~~~~~~~~~
 
+2.03
```

Comment:
According to the [schema issue](https://github.com/IATI/IATI-Schemas/issues/341), `iati-activity/sector/@vocabulary-uri` is updated, not this one (`iati-activity/transaction/recipient-region`)

Reply:
Above comment has been resolved in IATI/IATI-Schemas#411